src/components/loginForm/LoginForm.scss

```diff
@@ -0,0 +1,35 @@
+.login-form {
```

Comment:
I suggest to use only one time of naming convention for class with multiple words.
Either choose `.loginForm` or `.login-form`
The ideal is to stay consistent (I saw already few of them - one way and the other )